src/TeachABit.Repository/Repositories/Tecaj/ITecajeviRepository.cs

```diff
@@ -0,0 +1,10 @@
+namespace TeachABit.Repository.Repositories.Tecaj;
+using TeachABit.Model.Models.Tecaj;
+public interface ITecajeviRepository
+{
+    Task<List<Tecaj>> GetTecajList();
+    Task<Tecaj> GetTecaj(int id);
```

Comment:
Neka GetTecaj ovdje ima tip Task<Tecaj?> kako bi bilo isto kao što si stavio u repozitoriju.